[Jules]: Fix typescript build pipeline 2

### Task Requirements

You are a Senior Frontend Architect. Your task is to fix the TypeScript build pipeline for the Meraki HA panel by resolving module resolution and custom element declaration errors.

<detected_errors>
1. TS2307: Cannot find module './types/websocket' - The API contract file is not being resolved despite being in a subfolder.
2. TS2339: Property 'ha-...' does not exist on type 'JSX.IntrinsicElements' - Custom elements are not globally registered.
</detected_errors>

<instructions>
1. Check the project requirements to prevent regressions and update the requirements based on the fix.
2. Think step-by-step. We must ensure the physical type files exist, that the compiler is configured to search subdirectories, and that the custom elements are registered in the global namespace.
3. Action 1: Create (or update) `src/types/websocket.ts`. It must export the `WsCommand` enum and relevant interfaces.
4. Action 2: Create a global declaration file at `src/types/hass-elements.d.ts`. Use the `declare global` and `namespace JSX` pattern to register:
   - `ha-circular-progress`: { active?: boolean; alt?: string; className?: string }
   - `ha-settings-row`: { className?: string; children?: any; slot?: string }
   - `ha-icon`: { icon?: string; style?: any; className?: string; slot?: string }
   - `ha-card`: { header?: string; className?: string; children?: any }
   - `ha-tabs`: { scrollable?: boolean; "attr-for-selected"?: string; selected?: string; className?: string; children?: any }
   - `paper-tab`: { name?: string; className?: string; children?: any }
5. Action 3: Update `tsconfig.json`. Change `"include": ["src"]` to `"include": ["src/**/*"]` to ensure all subdirectories and declaration files are scanned.
6. Always output the complete merged files and proactively apply optimizations.
7. Output the code for `src/types/websocket.ts`, `src/types/hass-elements.d.ts`, and the updated `tsconfig.json`.
</instructions>

Please apply these changes to ensure the build pipeline can finally resolve the Meraki HA frontend types.

### Specific Constraints

Ignore all errors not related to the meraki_ha integration. Follow Home Assistant Sentence Case standards.

--- a/custom_components/meraki_ha/www/tsconfig.json
+++ b/custom_components/meraki_ha/www/tsconfig.json
@@ -18,6 +18,6 @@
     "noUnusedParameters": true,
     "noFallthroughCasesInSwitch": true
   },
-  "include": ["src"],
+  "include": ["src/**/*"],
   "references": [{ "path": "./tsconfig.node.json" }]
 }

